grumblr/app/controllers/application_controller.rb

```diff
@@ -2,4 +2,14 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-end
+
+    def set_access_control_headers
```

Comment:
we technically don't need these methods. That's only if we want to let other JS apps read our API. Since our JS is served from Rails, it's not needed here!